pom.xml

```diff
@@ -1942,11 +1942,11 @@
                 <scope>compile</scope>
             </dependency>
 
-            <dependency>
-              <groupId>com.sleepycat</groupId>
-              <artifactId>je</artifactId>
-              <version>18.3.12</version>
-            </dependency>
+<!--            <dependency>-->
+<!--              <groupId>com.sleepycat</groupId>-->
+<!--              <artifactId>je</artifactId>-->
+<!--              <version>18.3.12</version>-->
+<!--            </dependency>-->
```

Comment:
Has it been confirmed with @planetf1 @grahamwallis that this dependency can be removed from Egeria root POM?

Reply:
Good point. The entries under dependencyManagement get used not just to express a dependency for our code, but also they affect transitive dependencies (ie dependencies of dependencies of dependencies etc). Since we use Janus for our graph repo as well as for open lineage, I rather suspect we need them too. They allow us to control what version is used (ie in terms of security fixes)

You can run 'mvn dependency:tree' to see if we reference this artifact at all. If not, it's safe to remove. If it is (which I think it is), then it's not.

Thanks for spotting @GuyAckermans

Reply:
Looks like this may have caused the verify build failure